numpy/random/generator.pyx

```diff
@@ -3950,13 +3954,19 @@ cdef class Generator:
                [0, 1, 2],
                [3, 4, 5]])
 
+        >>> rng.permutation("abc")
+        Traceback (most recent call last):
+            ...
+        numpy.AxisError: x must be an integer or at least 1-dimensional
```

Comment:
Not sure an error example helps too much here, but maybe if someone thinks they can shuffle strings or similar objects.